fiatconnect-api.md

```diff
@@ -1796,6 +1802,47 @@ is below:
 
 * `'NG'`: The account number should be exactly 10 digits long, and only include the numbers 0-9.
 
+#### 7.3.2.2. `MobileMoney`
+
+Most of the mobile money's providers require only the phone number to process a transaction.  So, the best approach to make this schema general, is to add the *operator*.
+`Operator` represents the name of the mobile operator and `mobile` the phone number of the end-users with the country code (i.e., +225).
+
+```
+{
+  accountName: `string`,
+  institutionName: `string`,
+  mobile: `string`,
+  operator: `SupportedOperatorEnum`,
+  fiatAccountType: `FiatAccountTypeEnum.MobileMoney`
+}
+```
+
+##### 7.3.2.2.1. `SupportedOperatorEnum`
```

Comment:
I like the idea here, but rather than giving a potential list of providers just for Côte d'Ivoire, it might be better to just give a general list, and provide a mapping from enum values to actual providers. For example:

> Depending on the `allowedValues` field for `operator` in each country, the client SHOULD impose restrictions on the type of data the user can provide for the `operator` field. The values in the `allowedValues` field for `operator` MUST be from the `SupportedOperatorEnum` below.

```
[
  `ORANGE`,
  `MOOV`,
  `MTN`
]
```

> Each value in `SupportedOperatorEnum` corresponds to a specific mobile money provider. These are:

> * `ORANGE` corresponds to the [Orange Money](https://en.wikipedia.org/wiki/Orange_Money) provider
> * `MOOV` corresponds to the [Moov Money](https://moov-africa.bj/moov-money/) provider
> * `MTN` corresponds to the [MTN Mobile Money](https://www.worldremit.com/en/mtn-mobile-money) provider

Not sure if these links are correct, but hopefully you get the idea. Providers can select any subset of these to include in the `allowedValues` field for `operator` depending on what they support.